lib/writer.js

```diff
@@ -67,7 +67,7 @@ exports.markdown = function (version, commits, options) {
     Object.keys(this.types[type]).forEach(function (category) {
       var prefix = '*';
       var nested = types[type][category].length > 1;
-      var categoryHeading = '* **' + category + ':**';
+      var categoryHeading = prefix + (Boolean(category) ? ' **' + category + ':**' : '');
 
       if (nested) {
```

Comment:
Very good point! I've forgotten that an empty string can be still used as `types[type][category]`'s key 😆